docs/spark/tutorials/databricks-deployment.md

```diff
@@ -9,161 +9,211 @@ ms.custom: mvc
 
 # Deploy a .NET for Apache Spark application to Databricks
```

Comment:
The template for tutorials show that the word tutorial should be in the title and H1
https://review.docs.microsoft.com/en-us/help/contribute/global-tutorial-template?branch=master